pkg/gitreceive/k8s_util.go

```diff
@@ -241,7 +241,6 @@ func progress(msg string, interval time.Duration) chan bool {
 				return
 			case <-tick:
 				fmt.Println(msg)
-			default:
```

Comment:
got it. I agree this should be removed, but it's out of scope of this PR. can you submit a different PR for removing this?